scss/_variables.scss

```diff
@@ -119,6 +119,17 @@ $accent: #ff6358 !default;
 /// Used to provide contrast between the background and foreground colors.
 $accent-contrast: contrast-wcag( $accent ) !default;
 
+/// The color for informational messages and states.
+$info: #0058e9 !default;
+/// The color for success messages and states.
+$success: #37b400 !default;
+/// The color for warning messages and states.
+$warning: #ffc000 !default;
+/// The color for error messages and states.
+$error: #f31700 !default;
```

Comment:
Why should we move these? :)

Reply:
I just wanted to group the variables together e.g. first main theme color and then the rest of the colors.